mon-pix/tests/integration/components/certification-starter_test.js

```diff
@@ -188,51 +188,131 @@ module('Integration | Component | certification-starter', function (hooks) {
             accessCode: 'ABC123',
             sessionId: 123,
           });
-          sinon.assert.calledOnce(certificationCourse.save);
-          sinon.assert.calledOnce(resetStub);
+          sinon.assert.calledOnce(postMessageStub);
           sinon.assert.calledWithExactly(routerObserver.replaceWith, 'authenticated.certifications.resume', 456);
+
           assert.ok(true);
         });
       });
 
-      module('when the creation of certification course is in error', function () {
-        test('should display the appropriate error message when error status is 404', async function (assert) {
-          // given
-          const replaceWithStub = sinon.stub();
+      test('should not send a postMessage', async function (assert) {
```

Comment:
Il ne manquerait pas un module pour accompagner la description de ce test ?
On teste juste au dessus qu'un postmessage a été envoyé dans le même module que ce test là.

Reply:
Possiblement remettre `module('when the creation of certification course is in error', function () {`